### PR TITLE
Fixed NullpointerException

### PR DESCRIPTION
**Pull request by master117**
Fixed NullpointerException that occurred when an artist had only one page worth of content.

Made Input commands case-insensitive

### CODE REVIEW

**Comment by master117** [comment]
Fixes this issue: https://github.com/AvocadoFlour/KemonoDownloader/issues/6

File: KemonoDownloader/Logic/ProgramFunctions.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public static string ReadUserInput (string message, List<string> acceptableInput
             do
             {
                 Console.WriteLine(message);
-                string input = Console.ReadLine();
+                string input = Console.ReadLine().ToUpper();
```
Comment (AvocadoFlour) [inline KemonoDownloader/Logic/ProgramFunctions.cs:26]: Thank on this finesse.

File: KemonoDownloader/Program.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ static void Main(string[] args)
 
                 // How to save downloaded posts
                 Console.WriteLine("Do you want all of the media from a single post to also be put into post-based folder? \n");
-                choice = ReadUserInput("Input \"N\" for: artistname\\artworks file hierarcy. Input \"Y\" for: artistname\\post\\artworks file hierarcy.", new List<string>() { "y", "n" });
+                choice = ReadUserInput("Input \"N\" for: artistname\\artworks file hierarcy. Input \"Y\" for: artistname\\post\\artworks file hierarcy.", new List<string>() { "Y", "N" });
```
Comment (AvocadoFlour) [inline KemonoDownloader/Program.cs:52]: Thank on this finesse.

**Review by AvocadoFlour — APPROVED** [review]
I just encountered this today for the first time (the null exception when an artist has a single page).
It's cool to work together.

P.S. I hope you didn't receive an email every time I edited/made a comment..

**Pull request merged**